CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_zendesk v0.12.1
+
+## Bug Fixes
+- Updates the `int_zendesk__schedule_spine` model to convert the Holiday schedules into proper UTC values before being used in comparison with the schedule times. This ensures the holidays are properly mapped to schedules regardless of timezones.
```

Comment:
@fivetran-joemarkiewicz Since this is part of a batch release should we include a link to this specific PR and the changes introduced here?

Reply:
Yeah I will do the individual PR links for these since it is a batch release with multiple PRs.